mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -299,6 +299,8 @@ PopulateParamsXDL::isValidBlockwiseGemmXDLOPS(const InitParamsXDL &param,
     // clang-format off
     validWaveGemmSize = {
       std::make_tuple(32, 32, 2),
+      std::make_tuple(64, 64, 2),
+      std::make_tuple(128, 128, 2),
```

Comment:
Good job, thanks!

Reply:
No problem! Actually, I was thinking: now that we have reduction fully supported, should we remove having different sets for int8 and floating points? I mean we can try the same exact configurations on all the data types.

Reply:
Seems fair, yes we should give it a shot and see how much perf impact it is.

Though if there's not a lot then we should fall back to existing narrow pattern because then it filtered out a lot of tuning space.

Reply:
I will tune again and see if there is any improvement

Reply:
So yes, few int8 scenarios got a small improvement (7 to 11 %). 

I am attaching the performance comparison here (please, ignore f32 and f16, since they are exactly the same)

[enlarge_int8_space.zip](https://github.com/ROCmSoftwarePlatform/rocMLIR/files/10804246/enlarge_int8_space.zip)